app/models/booking.rb

```diff
@@ -1,4 +1,6 @@
 class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :product
+  enum status: [:available, :pending, :not_available]
+  validates :rating, inclusion: { in: (1..10)}
```

Comment:
Should we add presence: true to :rating?

Reply:
Some apps don't have ratings if there is only few ratings. I think it should be the average of some amount of reviews. Maybe we need to discuss about it.